frontend/src/components/skeletons/ApiKeySkelton.tsx

```
@@ -50,6 +50,7 @@ export function ApiKeysSkeleton() {
                 </tr>
               </thead>
               <tbody>
+                // NOSONAR As safe to use index as key - static skeleton items with fixed length
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)